### DISCUSSION TITLE
Is there anyway to use selenium with this.

### DISCUSSION THREAD

**Discussion by Deshdeepak1** [minbrowser/min#1969] [Ideas]
I like to automate few things with selenium and brave, but its too heavy. Anyway to use selenium with this?

**Comment by PalmerAL** [comment]
Currently no. Min is unlikely to provide much of a benefit in this case - the UI isn't a factor, and so Min is unlikely to be much faster than other Chromium-based browsers here. You could also consider using Electron directly.
